linux/libkmfl/debian/copyright

```diff
@@ -3,23 +3,25 @@ Upstream-Name: libkmfl
 Upstream-Contact: Doug Rintoul <doug_rintoul@sil.org>
 
 Files: *
-Copyright: Copyright (C) 2005 SIL International and Tavultesoft Pty Ltd
-License: GPL-2+
+Copyright: Copyright (C) 2017 SIL International
```

Comment:
Almost everywhere else on the PR, this line is left as
> Copyright (C) 2005  SIL International and Tavultesoft Pty Ltd

I wonder which should be used for consistency...
1. Copyright (C) 2005  SIL International and Tavultesoft Pty Ltd
2. Copyright (C) 2005, 2018 SIL International
3. Copyright (C) 2018 SIL International

Reply:
It should be `Copyright (C) 2005-2018 SIL International`.

Reply:
For clarification, do you want "Copyright (C) 2005-2018 SIL International" everywhere there is a copyright statement, or only in the debian/copyright and COPYING files?

Reply:
change made to debian/copyright in all 3 KMFL projects

Reply:
I think the fully copyright statement with full date range is probably only needed in the metadata. There's all sorts of complex argument about whether copyright applies to whole body of work of to components and I don't really want to get into it. If we just say `Copyright (C) SIL International` in headers I am comfortable with that.

Reply:
OK, I think this is good to go then

Reply:
I'm not sure about removing the years but made all of them SIL International only. Sure about wanting me to remove the years as well?

Reply:
No, not saying go and remove date ranges, but rather they are not as important in individual files.